tests/statsmodels/test_statsmodels_autolog.py

```diff
@@ -184,7 +184,9 @@ def test_statsmodels_autolog_respects_log_models_flag(log_models):
     run = get_latest_run()
     client = MlflowClient()
     artifact_paths = [artifact.path for artifact in client.list_artifacts(run.info.run_id)]
-    assert ("model" in artifact_paths) == log_models
+    assert (
```

Comment:
can we use `last_logged_model`?

Reply:
It did not work well, we probably need to clean up tthe hread local var.

Reply:
Let's add a fixture that calla `_reset_logged_model_id` in  `tests/conftest.py`.

Reply:
Added